packages/appstore/src/components/transaction-list/transaction-list.tsx

```diff
@@ -280,10 +313,8 @@ const FiatTransactionList = () => {
                         let icon_type = 'fiat';
                         let is_deriv_apps = false;
                         if (transaction.action_type === 'transfer') {
-                            const other_loginid =
-                                transaction.to?.loginid === loginid
-                                    ? transaction.from?.loginid
-                                    : transaction.to?.loginid;
+                            const other_party = transaction.to?.loginid === loginid ? transaction.from : transaction.to;
+                            const other_loginid = other_party?.loginid;
                             if (!other_loginid) return null;
                             const other_account = accounts[other_loginid];
```

Comment:
Would we still need this if we get `account_type`, `account_category` and `currency` in `statement` response?